src/sandstorm/supervisor.capnp

```diff
@@ -266,6 +266,24 @@ struct ApiTokenOwner {
 
       title @7 :Text;
       # Title as chosen by the user.
+
+      appTitle @8 :Util.LocalizedText;
+      # A copy of the app name for the corresponding UIView for presentation in the grain list.
+
+      lastUsed @9 :Int64;
+      # The last time the user used this API token with the associated grain, in milliseconds
+      # since the epoch (equivalent to javascript's new Date().getTime())
+
+      icon :group {
```

Comment:
It probably makes sense to define this as a separate struct rather than a group since it seems there's a decent chance we'll need to use it elsewhere. In fact it might even make sense to wrap all the new fields here in a struct called `DenormalizedGrainMetadata` or something like that.